components/list/list-item-drag-image.js

```diff
@@ -0,0 +1,126 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../inputs/input-checkbox.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodySmallStyles } from '../typography/styles.js';
+import { formatNumber } from '@brightspace-ui/intl/lib/number.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class ListItemDragImage extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * @ignore
+			 */
+			count: { type: Number }
+		};
+	}
+
+	static get styles() {
+		return [ bodySmallStyles, css`
+			:host {
+				display: block;
+				height: 70px;
+				left: -10000px;
```

Comment:
The `d2l-offscreen` styles could not be used because they mess up the `dataTransfer.setDragImage` result.